package.json

```diff
@@ -119,6 +119,7 @@
 		"cross-env": "6.0.3",
 		"cssnano": "4.1.10",
 		"deep-freeze": "0.0.1",
+		"eslint-plugin-import": "^2.22.1",
```

Comment:
This was a new dependency needed with the `@wordpress/scripts` update (or might be a side-effect of pulling in the updated `@wordpress/eslint-plugin` ruleset. It might be an artifact of new eslint requirements too.

Reply:
Speak of the devil: https://github.com/WordPress/gutenberg/pull/28525